docs/02-app/01-building-your-application/09-authentication/index.mdx

```diff
@@ -1,38 +1,443 @@
 ---
 title: Authentication
-description: Learn how to implement authentication in Next.js, covering best practices, securing routes, authorization techniques, and session management.
+description: Learn how to implement authentication in your Next.js application.
 ---
 
-To implement authentication in Next.js, familiarize yourself with three foundational concepts:
+Understanding authentication and authorization is crucial for protecting your application's data and ensuring a smooth user experience. This page will guide you through how to use Next.js features and third-party libraries to implement auth, as well as principles and best practices so you can choose the right strategy for your application.
 
-- **[Authentication](#authentication)** verifies if the user is who they say they are. It requires the user to prove their identity with something they have, such as a username and password.
-- **[Session Management](#session-management)** tracks the user's state (e.g. logged in) across multiple requests.
-- **[Authorization](#authorization)** decides what parts of the application the user is allowed to access.
+Before starting, it helps to break down the process into three key concepts, or steps:
```

Comment:
The word "steps" makes me think I have to do them in order, which I don't think we need to here. It's more for enumeration. 

```suggestion
Before starting, it helps to break down the process into three key concepts:
```